SemanticAnalysis/parser.y

```diff
@@ -339,9 +492,10 @@ param:                            data_type ID
                                     printf("param\n");
                                     
                                     default_value(type);
-                                    sym = makeSymbol($2, type, &value, 0, size, 'v', 1, 0);
+                                    s1 = popV();
+                                    sym = makeSymbol($2, s1->type, &value, s1->size, 'v', 1, 0);
                                     add_variable_to_table(sym);
-                                    $$ = makeNode(astParam, sym, $1, NULL, NULL, NULL);

```

Comment:
Are you sure this had to be removed? Don't we need to make it an ast node? If you are sure then some major changes will have to be done in `semantics.c`.

Reply:
I removed this by mistake.

Reply:
Resolved this.